src/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace System.Net
 {
-    internal enum CookieVariant
+    public enum CookieVariant
```

Comment:
Is this the change that you believe should have broken serializatoin for CoreCLR as well? Could you please check why the tst is apparently passing without it (ie. we have a test)?